File: ruby/URI_1044.rb
valores = gets.split

a = valores[0].to_i
b = valores[1].to_i

if (a > b) then
	if (a % b == 0) then
		puts "Sao Multiplos"
	else
		puts "Nao sao Multiplos"
	end
else
	if (b % a == 0) then
		puts "Sao Multiplos"
	else
		puts "Nao sao Multiplos"
	end
end

File: c-plus-plus/URI_1044.cpp
#include <iostream>
using namespace std;
int main () {
	int a, b;
	cin >> a >> b;
	bool multiple = false;
	if (a > b)
	{
		if (a % b == 0)
		{
			multiple = true;
		}
	}
	else
	{
		if (b % a == 0)
		{
			multiple = true;
		}
	}
	if (multiple)
	{
		cout << "Sao Multiplos" << endl;
	}
	else
	{
		cout << "Nao sao Multiplos" << endl;
	}
	
	return 0;
}
